webpack.config.js

```diff
@@ -16,5 +17,13 @@ module.exports = merge(defaultConfig, {
         plugin.constructor.name !== "DependencyExtractionWebpackPlugin"
     ),
     new DependencyExtractionWebpackPlugin(),
-  ]
+    new webpack.ProvidePlugin({
```

Comment:
This didnt seem to make it into main but we need to make sure this is included.
These changes are live in trunk:
https://plugins.trac.wordpress.org/browser/php-compatibility-checker/trunk/readme.txt#L89